db/migrations/versions/755b1be91ed5_.py

```diff
@@ -0,0 +1,52 @@
+"""empty message
+
+Revision ID: 755b1be91ed5
+Revises: 
+Create Date: 2022-08-15 16:55:27.754731
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '755b1be91ed5'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('applications',
+    sa.Column('id', sqlalchemy_utils.types.uuid.UUIDType(binary=False), nullable=False),
+    sa.Column('account_id', sa.String(), nullable=False),
+    sa.Column('round_id', sa.String(), nullable=False),
+    sa.Column('fund_id', sa.String(), nullable=False),
+    sa.Column('project_name', sa.String(), nullable=True),
+    sa.Column('started_at', sa.DateTime(), server_default=sa.text('now()'), nullable=True),
+    sa.Column('status', postgresql.ENUM('NOT_STARTED', 'IN_PROGRESS', 'SUBMITTED', 'COMPLETED', name='status'), nullable=False),
+    sa.Column('date_submitted', sa.DateTime(), nullable=True),
+    sa.Column('last_edited', sa.DateTime(), nullable=True),
```

Comment:
Would it be difficult to add a `created_at` column and populating? (would be useful to work out how long applications are taking from created to submitted)

Reply:
@gio-karunakaran Could add the started_at logic in this pr?

Reply:
Since its related to statuses

Reply:
I'm thinking of renaming started_at to created_at to make this clearer? will make this change if everyone is happy with that? @SentientHousePlant @gidsg :D

Reply:
Ah, OK I didn't see we had `started_at` already, so I think that's fine.

Reply:
👍🏾